polytope/polytope.py

```diff
@@ -1292,25 +1301,26 @@ def volume(polyreg):
         N = 50
     elif n == 2:
         N = 500
-    elif n ==3:
+    elif n == 3:
         N = 3000
     else:
         N = 10000
 
     l_b, u_b = polyreg.bounding_box
-    x = np.tile(l_b,(1,N)) +\
-        np.random.rand(n,N) *\
-        np.tile(u_b-l_b,(1,N) )
+    x = np.tile(l_b, (1, N)) +\
+        np.random.rand(n, N) *\
+        np.tile(u_b-l_b, (1, N))
     aux = np.dot(polyreg.A, x) -\
```

Comment:
What about the following variant:

```python
aux = (
    np.dot(polyreg.A, x) -
    np.tile(
        np.array([polyreg.b]).T,
        (1, N)))
```

Placing the operator `-` at the end of the line allows for aligning its two arguments. The closing parentheses have been placed at the end of the line in the spirit of using indentation to signify context (in contrast to braces, e.g. in C).